models/location-sql.js

```diff
@@ -0,0 +1,58 @@
+import db from '../db/connection.js'
+
+// Get all location data
+export async function getAllLocationData() {
+    const result = await db.query(`SELECT * FROM location;`);
+    return result.rows;
+}
+
+// Get location data by loc_id
+export async function getLocationByLocID(loc_id) {
+    const result = await db.query(`SELECT * FROM location WHERE loc_id = $1;`, [loc_id])
+    return result.rows;
+}
+
+// Get location data by media_id
+export async function getLocationByMedID(media_id) {
+    const result = await db.query(`SELECT * FROM location WHERE media_id = $1;`, [media_id])
+    return result.rows;
+}
+
+// Get location by lat lon
+export async function getLocationByLatLon(latitude, longitude) {
+    const result = await db.query(`SELECT * FROM location WHERE latitude = $1 AND longitude = $2;`, [latitude, longitude])
+    return result.rows;
+}
+
+// POST location
+export async function addLocData({media_id, country, city, postcode, street, latitude, longitude}) {
+    const result = await db.query( `INSERT INTO location (media_id, country, city, postcode, street, latitude, longitude) VALUES ($1, $2, $3, $4, $5, $6, $7);`,
```

Comment:
- change latitude/longitude to lat/lng
- media_id not required as loc_id will be stored in media table instead 
- only record location rather than country, city, postcode, street etc.